_config.yml

```diff
@@ -8,6 +8,15 @@ description: > # this means to ignore newlines until "baseurl:"
 
 twitter_username: jekyllrb
 github_username:  jekyll
+rss: rss
+dribbble_username: jekyll
+facebook_username: jekyll
+flickr_username: jekyll
+instagram_username: jekyll
+linkedin_username: jekyll
+pinterest_username: jekyll
+youtube_username: jekyll
+googleplus_username: +jekyll
 
```

Comment:
Good catch. I agree. I had done that originally, but it got reverted while testing.  Commenting them out again...